### DISCUSSION TITLE
Button shapes

### DISCUSSION THREAD

**Discussion by danonh** [chipzoller/hugo-clarity#203] [Q&A]
Hi, 
First of all thank you for great Hugo blog template, works really great and I can't wait to see version2.
 Is there a possibility to change shapes of buttons ("Tags" buttons, "Read more" buttons etc.) from rounded to sharpened? Also is there an option to modify buttons colors? 
Thanks,

**Comment by chipzoller** [comment]
There's always an option :) But we don't make a parameter to do this. This is one of those things where if you need/want it badly enough, you should probably fork it and make the modifications yourself. If you wanted to then submit a PR we'd consider pulling it in.
